athena-gcs/src/main/java/com/amazonaws/athena/connectors/gcs/GcsUtil.java

```diff
@@ -0,0 +1,160 @@
+                + " in the secret " + System.getenv(gcsCredentialKeysEnvVar));
+    }
+
+    /**
+     * Builds a string representation of an instance of {@link StorageSplit}
+     *
+     * @param splits An instance of {@link StorageSplit}
+     * @return String representation of an instance of {@link StorageSplit}
+     * @throws JsonProcessingException If JSON processing error happens
+     */
+    public static synchronized String splitAsJson(List<StorageSplit> splits) throws JsonProcessingException
+    {
+        return objectMapper.writeValueAsString(splits);
+    }
+
+    public static boolean isFieldTypeNull(Field field)
+    {
+        return field.getType() == null
+                || field.getType().equals(Types.MinorType.NULL.getType());
+    }
+
+    public static void installCaCertificate() throws IOException
+    {
+        ClassLoader classLoader = GcsRecordHandler.class.getClassLoader();
+        File file = new File(requireNonNull(classLoader.getResource("")).getFile());
+        File src = new File(file.getAbsolutePath() + File.separator + "cacert.pem");
+        File dest = new File(System.getenv(SSL_CERT_FILE_LOCATION));
+        File parentDest = new File(dest.getParent());
+        if (!dest.exists()) {
+            if (!parentDest.exists()) {
+                parentDest.mkdirs();
+            }
+            Files.copy(src.toPath(), dest.toPath(), StandardCopyOption.REPLACE_EXISTING);
+        }
+    }
+
+    public static void installGoogleCredentialsJsonFile() throws IOException
+    {
+        CachableSecretsManager secretsManager = new CachableSecretsManager(AWSSecretsManagerClientBuilder.defaultClient());
+        String gcsCredentialsJsonString = getGcsCredentialJsonString(secretsManager.getSecret(System.getenv(GCS_SECRET_KEY_ENV_VAR)), GCS_CREDENTIAL_KEYS_ENV_VAR);
+        File dest = new File(System.getenv(GOOGLE_SERVICE_ACCOUNT_JSON_TEMP_FILE_LOCATION));
+        boolean destDirExists = new File(dest.getParent()).mkdirs();
+        if (!destDirExists && dest.exists()) {
+            return;
+        }
+        try (OutputStream out = new FileOutputStream(dest)) {
+            out.write(gcsCredentialsJsonString.getBytes(StandardCharsets.UTF_8));
+            out.flush();
+        }
+    }
+
+    public static boolean isSupportedFileType(String classification)
+    {
+        return supportedTypeList.stream().anyMatch(str -> str.trim().equalsIgnoreCase(classification));
+    }
+
+    /**
+     * Returns the Datasource specific file format to be used to read a file (for retrieving schema or fetching data)
+     * @param format is classification from table param
+     * @return An instance of FileFormat
+     */
+    public static FileFormat getFileFormat(String format)
+    {
+        switch (format.toLowerCase()) {
+            case "parquet":
+                return FileFormat.PARQUET;
+            case "csv":
+                return FileFormat.CSV;
+        }
+        return null;
+    }
```

Comment:
This function doesn't need to exist.

Wherever you use this function, instead you can just use:
`FileFormat.valueOf(format.toUpperCase())`

Reply:
Removed the method and used FileFormat.valueOf(format.toUpperCase()) instead.